testsuite/MDAnalysisTests/utils/test_units.py

```diff
@@ -125,3 +125,12 @@ def test_unit_unconvertable(self):
         A = nm * 10.
         with pytest.raises(ValueError):
             units.convert(A, 'A', 'ps')
+
+    def test_MDANALYSIS_BASE_UNITS_correct(self):
+        ref = {"length": "A",
+               "time": "ps",
+               "energy": "kJ/mol",
+               "charge": "e",
+               "force": "kJ/(mol*A)",
+               "speed": "A/ps"}
+        assert ref == units.MDANALYSIS_BASE_UNITS
```

Comment:
This is essentially the same thing as flushing the units for coverage, it's not wrong, but it's also not really doing much. One way this could be improved could be to do a conversion check, say 1 nm == 10 * 1 * length?

Reply:
I suggested this test. Primarily I want to know when the dict is changed (either units added or altered).

If the actual entries are correct here in the references then we don't need to test them for correctness. 

The only thing I might suggest is to make `ref` a fixture and add a comment to make extra clear that this is a copy of the dict that we expect.